lib/view_model/wallet_keys_view_model.dart

```diff
@@ -172,7 +171,10 @@ abstract class WalletKeysViewModelBase with Store {
     final restoreHeightResult = await restoreHeight;
     return {
       if (_appStore.wallet!.seed != null) 'seed': _appStore.wallet!.seed!,
-      if (_appStore.wallet!.privateKey != null) 'private_key': _appStore.wallet!.privateKey!,
+      if (_appStore.wallet!.seed == null && _appStore.wallet!.hexSeed != null)
+        'hexSeed': _appStore.wallet!.hexSeed!,
```

Comment:
did you handle this case in restoring from QR?

Reply:
ah nice catch, I forgot to add this
latest pushes address this